Jenkinsfile

```
@@ -350,7 +350,7 @@ pipeline {
 					dir('eclipse.platform.swt') {
 						sh '''
 							mvn clean verify \
-								--batch-mode --threads 1C -V -U -e -DforkCount=0 \
+								--batch-mode --threads 1C -V -U -e -DforkCount=1 \
```

If I'm not mistake, the default is one. So just removing it would make it simpler.
```suggestion
								--batch-mode --threads 1C -V -U -e \
```

---

Trying it now.